doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,343 @@
+      "requests": {
+        "type": "object",
+        "description": "The lower bounds for resources that should be allocated to a connector pod",
+        "properties": {
+          "cpu": {
+            "type": "string",
+            "description": "Compute processing specified in units of Kubernetes CPUs as described in https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu"
+          },
+          "memory": {
+            "type": "string",
+            "description": "The number of bytes written as described in https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory"
+          }
+        }
+      },
+      "limits": {
+        "type": "object",
+        "description": "The upper bounds for resources that can be allocated to a connector pod",
+        "properties": {
+          "cpu": {
+            "type": "string",
+            "description": "Compute processing specified in units of Kubernetes CPUs as described in https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu"
+          },
+          "memory": {
+            "type": "string",
+            "description": "The number of bytes written as described in https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory"
+          }
+        }
+      }
+    }
+  },
+  "aioMetadata": {
+    "type": "object",
+    "description": "The AIO requirements to run this connector",
+    "properties": {
+      "aioMinVersion": {
+        "type": "string",
+        "description": "The minimal (inclusive) version that can run this connector."
+      },
+      "aioMaxVersion": {
+        "type": "string",
+        "description": "The maximal (inclusive) version that can run this connector."
+      }
+    }
+  },
+  "inboundEndpoints": {
+    "type": "array",
+    "description": "The array of inbound endpoints in a device that this connector will be deployed for.",
+    "items": [
+      {
+        "type": "object",
+        "properties": {
+          "endpointType": {
+            "type": "string",
+            "description": "The type of this inbound endpoint."
+          },
+          "configSchemas": {
+            "type": "object",
+            "description": "The inbound endpoint level JSON schemas for the various 'additional configuration' fields. A JSON schema must be provided for each 'additional configuration' only if that field is used by this connector.",
+            "properties": {
+              "additionalConfigurationSchema": {
+                "$ref": "#/definitions/schema"
+              }
+            }
+          },
+          "assets": {
+            "type": "array",
+            "description": "The array of assets that reference this inbound endpoint",
+            "items": [
+              {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
```

Comment:
What is this name for?

Reply:
asset name. I'll make this more explicit

Reply:
you're required to specify the asset names upfront? So we have an inbound endpoint type -> and then all assets that could be associated with that inbound endpoint type?